src/hipscat/io/paths.py

```diff
@@ -3,41 +3,64 @@
 import os
 
 
-def pixel_directory(catalog_path="", pixel_order=0, pixel_number=0):
+def pixel_directory(
+    catalog_path="", pixel_order=0, pixel_number=None, directory_number=None
+):
     """Create path *string* for a pixel directory. This will not create the directory.
 
-    The directory name will take the form of:
+    One of pixel_number or directory_number is required. The directory name will
+    take the HiPS standard form of:
 
-        <catalog_path>/Norder<pixel_order>/Npix<pixel_number>/
+        <catalog_path>/Norder=<pixel_order>/Dir=<directory number>
+
+    Where the directory number is calculated using integer division as:
+
+        (pixel_number/10000)*10000
 
     Args:
         catalog_path (str): base directory of the catalog (includes catalog name)
         pixel_order (int): the healpix order of the pixel
+        directory_number (int): directory number
         pixel_number (int): the healpix pixel
     Returns:
         string directory name
     """
-    return os.path.join(
-        catalog_path, f"Norder{int(pixel_order)}/Npix{int(pixel_number)}"
-    )
+    norder = int(pixel_order)
+    if pixel_number is None and directory_number is None:
+        raise ValueError(
+            "One of pixel_number or directory_number is required to create pixel directory"
+        )
+    if directory_number is not None:
+        ndir = directory_number
+    else:
+        npix = int(pixel_number)
+        ndir = int(npix / 10_000) * 10_000
+    return os.path.join(catalog_path, f"Norder={norder}/Dir={ndir}")
```

Comment:
If the constants are moved and there's no circular import issue, the string here could use the constants defined previously

Reply:
Same in the function below

Reply:
There's a circular import issue. =[

Reply:
Could the constants be moved to their own file to fix the circular import? I think it'd be nice to not have to change that.

Reply:
Oh the other constants are defined for the pixel info metadata column names. We could just define constants for the file names in this paths file then?